### ISSUE TITLE
geom_spaghetti break for Windows OS

### ISSUE THREAD

**Issue by julia-wrobel** [tidyfun/tidyfun#93]
There seems to be an error with geom_spaghetti on Windows machines but not on mac or linux (a student I'm working with ran into an error and I tested it on another windows machine and got the same error). It can be reproduced by the code below. It might be worth implementing appveyor along with travis to run automatic checks on a windows machine.

`library(tidyfun)

dti_df = tibble(
  id = refund::DTI$ID,
  visit = refund::DTI$visit,
  sex = refund::DTI$sex,
  case = factor(ifelse(refund::DTI$case, "MS", "control")))

dti_df$cca = tfd(refund::DTI$cca, arg = seq(0,1, l = 93))

dti_df %>%
  ggplot() +
  geom_spaghetti(aes(y = cca, col = case, alpha = .2 + .4*(case == "control"))) +
  facet_wrap(~ sex) +
  scale_alpha(guide = 'none', range = c(.2, .4))
`
produces error

> Error in .f(.x[[i]], ...) : object 'y___id' not found

**Comment by HerrMo** [comment]
I also get this error on a Linux  Mint Cinamon 19.2 machine (and additional warning messages) using R 3.6.1 and, for example, 
```
library(tidyfun)
library(ggplot2)
library(dplyr)

chf_df %>% 
  filter(id == 1) %>% 
  ggplot(aes(y = activity)) + 
  geom_spaghetti() 
```
When I narrowed it down, I got this far:

In
https://github.com/tidyfun/tidyfun/blob/aee6465167bf01d88de10aa5fd4b15b9643dabfb/R/geom-spaghetti.R#L53-L55

there is no column named `y___id` (anymore?).
```
tf_unnest(data, y, .arg = params$arg, .sep = "___") %>%
                      select(-group)
```
produces 

>  A tibble: 10,080 x 3
>  y___arg y___value PANEL

Using 

```
compute_layer = function(self, data, params, layout) {
  stopifnot(is_tf(pull(data, y)))
  tf_eval <-
    suppressMessages(tf_unnest(data, y, .arg = params$arg, .sep = "___")) %>%
      # select(-group) %>%
      rename(x = y___arg, y = y___value)
      tf_eval
},
```
reproduces the plot - at least with a different theme and different line colour.

**Comment by fabian-s** [comment]
thx, this was due to a [breaking change](https://cran.r-project.org/web/packages/tidyr/news/news.html) in `tidyr::unnest` behavior introduced in tidyr 1.0.0, 
fixed in 85083c903844d9109e1d40cbff99e5fa5ddf09d2.
@julia-wrobel, @HerrMo : should disappear if you `remotes::install_github("tidyfun/tidyfun", ref = "dev")` (or you go back to `remotes::install_version("tidyr", version = "0.8.3")`, but that might break other tibblyverse packages, who really knows what these people are up to....)  

also: adding your session info to bug reports makes it much easier to reproduce/understand -- please do use `{reprex}` / add `sessioninfo::session_info()`
